<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonSettingsWidget</class>
 <widget class="QWidget" name="CommonSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>492</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBoxTextProperties">
     <property name="title">
      <string>Text Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButtonTextFont">
          <property name="text">
           <string>Font...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelFont">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="toolButtonTextColor">
          <property name="text">
           <string>Text Color...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicViewTextColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonShadowColor">
          <property name="text">
           <string>Shadow Color...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="graphicViewShadowColor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="labelTextAlign">
          <property name="text">
           <string>Alignment:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxVerticalAling">
          <item>
           <property name="text">
            <string>Top</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingTop.png</normaloff>:/icons/icons/alingTop.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Middle</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingMiddle.png</normaloff>:/icons/icons/alingMiddle.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bottom</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingBottom.png</normaloff>:/icons/icons/alingBottom.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxHorizontalAling">
          <item>
           <property name="text">
            <string>Left</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingLeft.png</normaloff>:/icons/icons/alingLeft.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingCenter.png</normaloff>:/icons/icons/alingCenter.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
           <property name="icon">
            <iconset resource="softprojector.qrc">
             <normaloff>:/icons/icons/alingRight.png</normaloff>:/icons/icons/alingRight.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="labelTextAlign_2">
       <property name="text">
        <string>Animation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTransitionType">
       <item>
        <property name="text">
         <string>None</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranNone.png</normaloff>:/icons/icons/tranNone.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fade</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranFade.png</normaloff>:/icons/icons/tranFade.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fade Out-In</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranFadeOutIn.png</normaloff>:/icons/icons/tranFadeOutIn.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move Right</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranMoveRight.png</normaloff>:/icons/icons/tranMoveRight.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move Left</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranMoveLeft.png</normaloff>:/icons/icons/tranMoveLeft.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move Up</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranMoveUp.png</normaloff>:/icons/icons/tranMoveUp.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Move Down</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/tranMoveDown.png</normaloff>:/icons/icons/tranMoveDown.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTextAlign_3">
       <property name="text">
        <string>Text Effects:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTextEffects">
       <item>
        <property name="text">
         <string>None</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/effectsNone.png</normaloff>:/icons/icons/effectsNone.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shadow</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/effectsShadow.png</normaloff>:/icons/icons/effectsShadow.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blurred Shadow</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/effectsBlurredShadow.png</normaloff>:/icons/icons/effectsBlurredShadow.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelTextAlign_4">
       <property name="text">
        <string>Background:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxBackgoundType">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Solid Color</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/solidColor.png</normaloff>:/icons/icons/solidColor.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Picture</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/photo.png</normaloff>:/icons/icons/photo.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Video</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/video.png</normaloff>:/icons/icons/video.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsViewBackgroundColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditBackPath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowseBackground">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelMaxPercent">
       <property name="text">
        <string>Amount of screen to use:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxMaxScreen">
       <property name="toolTip">
        <string>Percent of screen to be used.</string>
       </property>
       <property name="suffix">
        <string notr="true">%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_MaxTopBottom">
       <property name="text">
        <string>Position To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxScreenPosition">
       <property name="toolTip">
        <string>Select to use either top portion of the screen or bottom.</string>
       </property>
       <item>
        <property name="text">
         <string>Top of Screen</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/toTopOfScreen.png</normaloff>:/icons/icons/toTopOfScreen.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Botton of Screen</string>
        </property>
        <property name="icon">
         <iconset resource="softprojector.qrc">
          <normaloff>:/icons/icons/toBottomOfScreen.png</normaloff>:/icons/icons/toBottomOfScreen.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="pushButtonDefault">
       <property name="text">
        <string>Reset All To Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>

